core-bundle/src/Resources/contao/modules/ModuleSearch.php

```diff
@@ -56,19 +56,12 @@ public function generate()
 	 */
 	protected function compile()
 	{
-		// Mark the x and y parameter as used (see #4277)
-		if (isset($_GET['x']))
-		{
-			Input::get('x');
-			Input::get('y');
```

Comment:
Using a `<form>` that sends a GET request will also add the `&x=` and `&y=` parameters to the query string. Did you remove this because we no longer check the query string but only the path?

Reply:
Not worth the effort IMHO.